cpu/lpc2387/include/cpu_conf.h

```diff
@@ -79,8 +79,7 @@ extern "C" {
  * @brief   Stack size used for the exception (ISR) stack
  * @{
  */
-extern unsigned __stack_irq_size;
-#define ISR_STACKSIZE                   ((unsigned) &__stack_irq_size)
+#define ISR_STACKSIZE                   (400)
```

Comment:
Maybe a comment to point to the linker file would make sense, so that those values never get out of sync. I wonder if it wouldn't be possible to allocate the stack from within in C, so that the linker script does not need to have this value hardcoded.

Reply:
yea the addresses of the interrupt stacks are set in `lpc2387/startup.s`, so they might as well be at an arbitrary location.

Reply:
Mind to either add the comment or allocate it from C?

Being honest: That value has not been touched for ages, so I believe a hardcoded value is just fine here. But having a comment in case some miracle happens and someone wants to change it will not hurt.

Reply:
I've moved the allocations to C code.